### PR TITLE
Add afterparty to the schedule

### PR DESCRIPTION
**Pull request by anellis**
People won't show up if they can't plan to be there!

### CODE REVIEW

File: source/views/schedule.html.haml [hunk 1]
```diff
@@ -49,3 +49,11 @@ title: Schedule - RailsBridge Triangle
           %h4.timeline-title Wind-down
           %p
             We'll talk about where you can go from here and say our goodbyes.
+
+      %li
+        %span.timeline-time 05:00 pm - ? (optional)
+        .timeline-description
+          %h4.timeline-title Afterparty
+          %p
+            Head to a local bar or restaurant, get some drink and appetizers,
```
Comment (drapergeek) [inline source/views/schedule.html.haml]: Get some drinks maybe?

Comment (anellis) [inline source/views/schedule.html.haml]: No. One drink. That everyone can share.

File: source/views/schedule.html.haml [hunk 2]
```diff
@@ -49,3 +49,11 @@ title: Schedule - RailsBridge Triangle
...
+            and celebrate all the awesome new things you've learned!
```
Comment (drapergeek) [inline source/views/schedule.html.haml]: Maybe add something like:

This is a great chance to ask your TAs about what it's like to work as a developer. I'm sure they'd love to answer any questions you can throw at them.

or ya know, something worded better

**Comment by anellis** [comment]
@drapergeek Updated

**Comment by anellis** [comment]
Closed with 041272f

**Pull request closed without merging**